<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
  </property>
  <widget class="QTextBrowser" name="text_note">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 11pt &quot;等线 Light&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'等线 Light'; font-size:11pt; font-weight:24; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;注册须知：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;您好！欢迎使用我们的图书馆管理系统，您只需要简单的注册便可以开启知识之旅，但在注册之前，请仔细阅读一下条文。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400;&quot;&gt;以上便是您需要遵循的使用规则，同意条文，才可继续注册！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>基本信息：</string>
   </property>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string> *姓名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string> *性别</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_type">
     <property name="text">
      <string> *身份</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_account">
     <property name="text">
      <string> *账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_key">
     <property name="text">
      <string> *密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>133</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_gender">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="combox_identify">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
     <item>
      <property name="text">
       <string>学生</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>管理员</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>教职工</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_account">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_key">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/background/resourse/b_W.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>splitter_4</zorder>
   <zorder>splitter</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_user">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>详细信息</string>
   </property>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_number"/>
    <widget class="QLineEdit" name="lineEdit_class"/>
    <widget class="QComboBox" name="comboBox_college">
     <item>
      <property name="text">
       <string>计算机学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>信息与通信工程学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>电子工程学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自动化学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>软件学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>经济管理学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人文学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>理学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>数字传媒学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>国际学院</string>
      </property>
     </item>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_regyear">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
     <property name="minimumDate">
      <date>
       <year>1960</year>
       <month>9</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>66</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_index">
     <property name="text">
      <string>   *学号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_class">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>   *班级</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_college">
     <property name="text">
      <string>   *学院</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_regyear">
     <property name="styleSheet">
      <string notr="true">font: 5pt &quot;ADMUI3Lg&quot;;</string>
     </property>
     <property name="text">
      <string>*入学时间</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>用户照片</string>
   </property>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>124</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>124</height>
     </size>
    </property>
    <property name="text">
     <string>     上传照片</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>118</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>118</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove rgb(255,255,255,0,0);border-radius:3px;padding:2px 2px;
background-image: url(:/background/resourse/button.png);
font: 6pt &quot;ADMUI3Lg&quot;;
background-color: rgb(47, 92, 255);
color:white;</string>
    </property>
    <property name="text">
     <string>上传图片</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_reg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>84</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>125</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 14pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	background-image: url(:/background/resourse/button.png);
}
QPushButton:hover{
	background-image: url(:/background/resourse/button_h.png);
}
QPushButton:pressed{	
	background-image: url(:/background/resourse/button_P.png);
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>125</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QRadioButton" name="Button_accept">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>同意</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Button_reject">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>拒绝</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_admin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>详细信息</string>
   </property>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_number_admin">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_code_admin">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_college_admin">
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:4px;padding:2px 2px;</string>
     </property>
     <item>
      <property name="text">
       <string>计算机学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>信息与通信工程学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>电子工程学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自动化学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>软件学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>经济管理学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人文学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>理学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>数字传媒学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>国际学院</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>   *职工号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>   *验证码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="styleSheet">
      <string notr="true">font: 5pt &quot;ADMUI3Lg&quot;;</string>
     </property>
     <property name="text">
      <string>*所属单位</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>506</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/background/resourse/register.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>text_note</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_user</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>Button_reg</zorder>
  <zorder>splitter_2</zorder>
  <zorder>groupBox_admin</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
